### PR TITLE
Minor fixes 

### PR DESCRIPTION
**Pull request by ashurakova**

### CODE REVIEW

File: SeaBattleLibrary/Ships/OneDeskShip.cs [hunk 1]
```diff
@@ -6,22 +6,20 @@ namespace SeaBattleLibrary
     public class OneDeskShip : Ship
     {
         ShipPlacementValidation OneDeskShipValidation = new ShipPlacementValidation();
-        public override void ShipPlacement(int n)
+        public override void ShipPlacement(int ShipsToPlace)
```
Comment (HeorhiSemianeniaiTechArt) [inline SeaBattleLibrary/Ships/OneDeskShip.cs]: All parameters in C# should be in camel case (start from low letter) base on code style. Please review the whole application.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

File: SeaBattle/Program.cs [hunk 2]
```diff
@@ -18,19 +18,19 @@ static void Main(string[] args)
             OneDeskShip.ShipPlacement(4);
             ShipPlacementValidation.FullShipsPlacementValidation(BattleShip.BotField);
             User user = new User();
-            Boolean GameFinished = false;
-            while (GameFinished != true)
+            Boolean IsGameFinished = false;
```
Comment (HeorhiSemianeniaiTechArt) [inline SeaBattle/Program.cs]: please change all Boolean to bool. The second implementation is more readable.
It is no different but it is good practice to use Boolean or bool only.
https://stackoverflow.com/questions/134746/what-is-the-difference-between-bool-and-boolean-types-in-c-sharp

File: SeaBattleLibrary/Ships/OneDeskShip.cs (hunk 1 above)
Comment (ashurakova) [reply SeaBattleLibrary/Ships/OneDeskShip.cs]: Fixed.

File: SeaBattle/Program.cs (hunk 2 above)
Comment (ashurakova) [reply SeaBattle/Program.cs]: Fixed.

File: SeaBattleLibrary/Ships/TwoDeskShip.cs [hunk 3]
```diff
@@ -6,30 +6,28 @@ namespace SeaBattleLibrary
     public class TwoDeskShip : Ship
     {
         ShipPlacementValidation TwoDeskShipValidation = new ShipPlacementValidation();
-        public override void ShipPlacement(int n)
+        public override void ShipPlacement(int shipsToPlace)
         {
-            int q = 0;
-            while (q < n)
+            int PlacedShips = 0;
```
Comment (HeorhiSemianeniaiTechArt) [inline SeaBattleLibrary/Ships/TwoDeskShip.cs]: The same started from the lower case.

**Pull request merged**